webapp/channels/src/components/sidebar_right_menu/sidebar_right_menu.tsx

```diff
@@ -2,76 +2,70 @@
 // See LICENSE.txt for license information.
 
 import classNames from 'classnames';
-import React from 'react';
+import React, {memo} from 'react';
 import {Link} from 'react-router-dom';
 import {CSSTransition} from 'react-transition-group';
 
-import * as GlobalActions from 'actions/global_actions';
-
 import MainMenu from 'components/main_menu';
 
 import {Constants} from 'utils/constants';
 
-type Action = {
-    openRhsMenu: () => void;
-}
+import {ANIMATION_DURATION} from './constant';
 
 type Props = {
     isMobileView: boolean;
     isOpen: boolean;
     teamDisplayName?: string;
     siteName?: string;
-    actions: Action;
```

Comment:
On `webapp/channels/src/components/sidebar_right_menu/index.ts` we are still defining the actions, which are not needed. Can you remove them from there?